web/app/utils/hermes-urls.d.ts

```diff
@@ -1,5 +0,0 @@
-declare module "hermes/utils/hermes-urls" {
-  export const HERMES_GITHUB_REPO_URL: string;
-  export const TEST_SUPPORT_URL: string;
-  export const TEST_SHORT_LINK_BASE_URL: string;
-}
```

Comment:
I shouldn't have been trying to export `const`s from a `d.ts` in the first place.

Reply:
I made this to appease Mirage but it turns out not to need it.